lib/web_ui/lib/src/engine/vector_math.dart

```diff
@@ -432,6 +432,27 @@ class Matrix4 {
       _m4storage[14] == 0.0 &&
       _m4storage[15] == 1.0;
 
+  /// Whether transform is identity or simple translation using m[12,13,14].
+  ///
+  /// We check for [15] first since that will eliminate bounds checks for rest.
+  bool isIdentityOrTranslation() =>
+      _m4storage[15] == 1.0 &&
+      _m4storage[0] == 1.0 // col 1
+          &&
```

Comment:
This doesn't look right. Maybe move `&&` before `// col 1`?

Reply:
Done.